features/admin.applications.v1/components/forms/general-details-form.tsx

```diff
@@ -378,7 +388,8 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
                             ".placeholder")
                     }
                     value={ name }
-                    readOnly={ readOnly || orgType === OrganizationType.SUBORGANIZATION }
+                    hidden={ isSubOrganizationType }
```

Comment:
rather than hiding one by one, let's hide the whole thing using the config

Reply:
This didn't work for me.
I think the thing is we are using Form here and the fields need to be immediately after the Form tag and when a Fragment is added it doesn't render as expected..Not sure if there's a different solution here..

Reply:
This will be fixed in a the PR with the phase 2 implementation